### PR TITLE
[ci skip] Add new maintainer

### PR DESCRIPTION
**Pull request by glemaitre**
@jakirkham Here we go. Thanks

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,3 +33,5 @@ about:
 extra:
   recipe-maintainers:
     - arokem
+    - glemaitre
+    - darcymason
```
Comment (jakirkham) [inline recipe/meta.yaml:37]: Just to confirm, are you ok being added as a maintainer, @darcymason?

Comment (darcymason) [reply recipe/meta.yaml:37]: Yes

**Comment by jakirkham** [comment]
Thanks all. Should be good to go. Please let us know if you run into any issues.

**Comment by glemaitre** [comment]
Thanks I can see the green button. Let's prepare the release then :).

**Pull request merged**